recipes/gemspa-spt/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "gemspa-spt" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gemspa-spt-{{ version }}.tar.gz
+  sha256: c45971d25859a4d2e013f1cedb692b36f3946151f4d492eb5d356fd327442cce
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - numpy
+    - pandas
+    - python >=3.8
+    - scikit-image
```

```suggestion
    - scipy
```

I feel the author meant [`scipy`](https://github.com/liamholtlab/gemspa-spt/blob/0032d5c61e72abe2dcbf817ba8215ef6f36426f0/src/gemspa_spt/particle_tracks.py#L15), not skimage.

---

See: https://github.com/liamholtlab/gemspa-spt/issues/1